{
    "python.pythonPath": "c:\\Users\\ederi\\virtualback4\\Scripts\\python.exe",
    "python.linting.pylintArgs": [
        "--load-plugins=pylint_django, pylint_celery" // para evitar que me de el problema en el objeto de los modelos de django
    ]
}